src/lib/components/connect-button/connect-button.svelte

```diff
@@ -1,10 +1,31 @@
 <script lang="ts">
   import Button from 'radicle-design-system/Button.svelte';
-  import walletStore from '$lib/stores/wallet';
+  import wallet from '$lib/stores/wallet';
+  import ens from '$lib/stores/ens';
+
+  async function connect() {
```

Comment:
Question for the future: do we prefer "functions" or const myFunc = () => {} format?

Reply:
I don't have a strong preference either way, do you?

Reply:
No, no strong opinion. I am used to `const`, but let's keep it as is; no reason to change.
To know so that we are consistent.